### PR TITLE
Request Enumerables Solution Review

### PR DESCRIPTION
**Pull request by Gnwin**
Advanced Enumerables Solution.
We built some of ruby's own enumerables like:

my_each
my_each_with_index
my_select
my_all?
e. t. c...

### CODE REVIEW

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Invalid Submission :x: :disappointed: :stop_sign: 
## Hello @Gnwin and @pacyL2K19   :handshake: :handshake: :handshake: 

Thank you for submitting a code review request. And congratulations on that you have completed the enumerable methods project. However your code review request is marked as invalid due to the following reasons:

1. The submitted Pull Request contains only the changes in readme file, => this means you should have submitted the Pull Request which contains actual enumerable methods that you wrote.
2. I can see that your project already has enumerable methods implemented but looks like you worked directly in the `develop branch`, that is a violation to `the correct git/github flow` hence we cannot review it.
3. You should have worked on the feature branch, you will merge the code to develop branch after TSEs approve your project, don't ever work directly in develop branch.

_`Action to be taken:` While you are preparing for the other code review request submission, we need you to submit an open Pull Request which contains the actual codes you wrote, I mean the Pull Request in which you wrote the all of the enumerables, and it needs to be open => (not merged and not closed)_

Happy coding!!!
___
_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

**Comment by pacyL2K19** [comment]
Hello Dear, I am not sure to understand that you are not able to see our enumerable file because after checking I saw that the file actually exists

**Comment by descholar-ceo** [comment]
> Hello Dear, I am not sure to understand that you are not able to see our enumerable file because after checking I saw that the file actually exists

Hello @pacyL2K19 Thank you for reaching out for clarifictaion, actually the problem is not the file, the file is there and I saw aal of your enumerables, the problem is that the Pull Request you submitted doesn't have commits in which you worked on those enumerables, so kindly make sure that whenever you submit a code review request, make sure that PR you submit contains the commits in which you have worked on the project you are submitting, otherwise you would be violating git/github flow

**Pull request closed without merging**
